modular_zubbers/code/modules/quirks/code/positive_quirks/telepathy.dm

```diff
@@ -0,0 +1,178 @@
+/datum/mutation/human/telepathy
+	power_path = /datum/action/cooldown/spell/pointed/telepathy
+
+/datum/action/cooldown/spell/pointed/telepathy
+	name = "Telepathic Communication"
+	desc = "<b>Left click</b>: point target to project a thought to them. <b>Right click</b>: project to your last thought target, if in range."
+	button_icon = 'icons/mob/actions/actions_revenant.dmi'
+	button_icon_state = "r_transmit"
+	spell_requirements = SPELL_REQUIRES_NO_ANTIMAGIC
+	antimagic_flags = MAGIC_RESISTANCE_MIND
+	cooldown_time = 1 SECONDS
+	cast_range = 7
+	/// What's the last mob we point-targeted with this ability?
+	var/datum/weakref/last_target_ref
+	/// The message we send
+	var/message
+	/// Are we blocking casts?
+	var/blocked = FALSE
```

Comment:
Ill be completely honest reworking telepathy into pointed is something I think even tg would like

Reply:
Definitely agree. Might take this upstream after